.github/workflows/build-and-preview-site.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       - name: Install and Build 🔧 
         run: |
-          npm install
+          npm install --legacy-peer-deps
```

Comment:
reason?

Reply:
I have included this because some packages do not support React 18. Therefore, during the installation process, it will skip those packages.

Reply:
but is it safe?

Reply:
It is not permanent change we will revert this when all the packages which currently do not support React - 18 are replaced. It is just to bypass those dependencies. I can build the site successfully after these changes. Also major dependency is Meshery-embed-design package i think when it will be upgraded we can remove this.